example/testcontent/models.py

```diff
@@ -109,3 +109,8 @@ def get_absolute_url(self):
 
 class TestLiveBlog(Content, AbstractLiveBlog):
     """Example LiveBlog implementation"""
+
+    class Mapping(Content.Mapping):
```

Comment:
yeah, this will overwrite each base contents mapping (`class Mapping(BaseContent.Mapping):`)